src/Setup/DevDivVsix/PortableFacades/PortableFacades.swr

```diff
@@ -1,29 +1,30 @@
 use vs
 
 package name=PortableFacades
-        version=1.2.0.0
+        version=1.3.0.0
```

Comment:
@jasonmalinowski, @DustinCampbell another area to consider.  If I understand this correctly it's effectively removing the 1.0 .NET facades from our insertion.  Hence any component that depends on them will need to be migrated to the new version @tmat is adding. 

I truthfully don't understand how this works.  But I'm concerned others that depend on the old package will need to go through a similar transformation as Roslyn is on this PR.